experimental/web_dashboard/lib/src/widgets/dialogs.dart

```diff
@@ -11,10 +11,12 @@ import '../app.dart';
 import 'edit_entry.dart';
 
 class NewCategoryDialog extends StatelessWidget {
+  const NewCategoryDialog({Key key}) : super(key: key);
+
   @override
   Widget build(BuildContext context) {
-    return SimpleDialog(
-      title: const Text('New Category'),
+    return const SimpleDialog(
+      title: Text('New Category'),
       children: <Widget>[
```

Comment:
I wish we had a lint just for these unnecessary `<Widget>` specifiers that used to be required.

Reply:
It doesn't look like anyone's filed an issue for this yet: https://github.com/flutter/flutter/labels/p%3A%20flutter_lints